### PR TITLE
clean up/reorg maven config, inheriting from fcrepo-parent

### PR DESCRIPTION
**Pull request by acoburn**
See: https://jira.duraspace.org/browse/FCREPO-1728

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -746,116 +482,30 @@
...
-          <outputName>surefire-report</outputName>
-          <aggregate>true</aggregate>
-          <reportsDirectories>
-            <reportsDirectory>${project.build.directory}/surefire-reports/</reportsDirectory>
-            <reportsDirectory>${project.build.directory}/failsafe-reports/</reportsDirectory>
-          </reportsDirectories>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-changelog-plugin</artifactId>
```
Comment (awoods) [inline pom.xml]: We want to keep this plugin, minus the `<configuration>` block.

Comment (acoburn) [inline pom.xml]: Re-added the changelog plugin

File: pom.xml [hunk 2]
```diff
@@ -759,116 +499,30 @@
...
-              <report>property-updates-report</report>
-            </reports>
-          </reportSet>
-        </reportSets>
       </plugin>
     </plugins>
   </reporting>
 
-
-  <profiles>
```
Comment (awoods) [inline pom.xml]: Running: `mvn help:effective-pom`, it appears that `fcrepo-camel-webapp/pom.xml` "profiles" overwrites the inheritance of these release-related profiles, which likely argues for not removing them from here until we drop `fcrepo-camel-webapp`.

Comment (acoburn) [inline pom.xml]: See my other note -- the profiles are not being overwritten. Inherited profiles are not displayed in `mvn help:effective-pom`, but they are when you run `mvn help:all-profiles`. With that second command you will see that both `release-sign-artifacts` and `sonatype-oss-release` are present for all modules but inactive.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/commit/a774c5dcdddcdce8374d613f4d8dd4d011303b3e

**Pull request closed without merging**